loopchain/blockchain/invoke_result.py

```diff
@@ -98,6 +98,23 @@ def _extract_validators(self, prev_block_votes: List[List[str]]) -> List[str]:
         else:
             return []
 
+    @classmethod
+    def from_block(cls, block: 'Block'):
+        header: 'BlockHeader' = block.header
+        body: 'BlockBody' = block.body
+
+        return cls(
+            height=header.height,
+            transactions=body.transactions,
+            prev_peer_id=header.peer_id,
+            block_hash=header.hash,
+            prev_block_hash=header.prev_hash,
+            timestamp=header.timestamp,
+            prev_votes=body.prev_votes,
+            tx_versioner=TransactionVersioner(),
```

Comment:
TrasnactionVersioner must be composed in configuration.
It means that this has to use versionoer in Blockchain object.

Reply:
Thank you. I understood.

I will apply your feedbacks right away.